modules/web/src/app/cluster/details/cluster/rbac/component.ts

```diff
@@ -37,44 +39,33 @@ import {isClusterRunning} from '@shared/utils/health-status';
 export class RBACComponent implements OnInit, OnDestroy {
   @Input() cluster: Cluster;
   @Input() projectID: string;
+
+  readonly RBACKind = Kind;
+  modes: Kind[] = [Kind.ServiceAccount, Kind.User, Kind.Group];
+  modeControl = new FormControl<Kind>(Kind.ServiceAccount);
+
   isClusterRunning: boolean;
-  clusterBindingsDataSource = new MatTableDataSource<SimpleClusterBinding>();
-  clusterBindingsDisplayedColumns: string[] = ['kind', 'name', 'clusterRole', 'actions'];
-  bindingsDataSource = new MatTableDataSource<SimpleBinding>();
-  bindingsDisplayedColumns: string[] = ['kind', 'name', 'clusterRole', 'namespace', 'actions'];
-  private readonly _refreshTime = 30;
+  private readonly _refreshTime = 10;
   private readonly _refreshTimer$ = timer(0, this._appConfigService.getRefreshTimeBase() * this._refreshTime);
-  private _refresh = new Subject<void>();
   private _unsubscribe = new Subject<void>();
 
   constructor(
     private readonly _clusterService: ClusterService,
     private readonly _rbacService: RBACService,
+    private readonly _clusterServiceAccountService: ClusterServiceAccountService,
     private readonly _matDialog: MatDialog,
     private readonly _notificationService: NotificationService,
     private readonly _appConfigService: AppConfigService
   ) {}
 
   ngOnInit(): void {
-    merge(this._refreshTimer$, this._refresh)
+    this._refreshTimer$
       .pipe(
         switchMap(_ => this._clusterService.health(this.projectID, this.cluster.id)),
```

Comment:
I think we should take this health value from parent component because adding it here causes 2 health API requests (one from parent component and one here). Wdyt?